packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -53,10 +59,14 @@ class RenderAndroidView extends RenderBox {
   RenderAndroidView({
     @required AndroidViewController viewController,
     @required this.hitTestBehavior,
+    List<OneSequenceGestureRecognizer> gestureRecognizers = const <OneSequenceGestureRecognizer> [],
```

Comment:
Maybe call this forwardingGestureRecognizers? Since (if I understand the docs right) unlike all other gesture recognizers, nothing is responded to in dart